aws-gitlab-main.yml

```diff
@@ -75,17 +75,60 @@ Parameters:
   GitLabDBMasterUserPasswordParameter:
     Type: String
     Description: Password for the master user. At least 8 characters. No slash, at symbol, or parentheses. 
+    NoEcho: True 
 
   GitLabAllocatedStorageParameter:
     Type: String
     Description: Amount of storage in gigabytes to be allocated for the database intsance
     Default: '100' #minimum for PostgreSQL 
 
   #ELASTICACHE/REDIS
-  CacheNodeTypeParameter:
+  GitLabCacheNodeTypeParameter:
     Type: String
     Description: Node type;  Multi-AZ not supported on T1 instances
     Default: cache.t3.medium 
+  
+  #BASTION HOSTS
+  GitLabBastionSecurityGroupCIDRParameter:
+    Type: String
+    Description: Enable SSH access from what IP address(es). For stricter security, specify a single IP 
+    Default: 0.0.0.0/0 
```

Comment:
This should be private cidr of whoever will be accessing gitlab. 0s are everybody
Make this default 127.0.0.1 (loopback) and instruct user to change it after deployment.... This should be your public IP when you deploy